### DISCUSSION TITLE
3.49.0

### DISCUSSION THREAD

**Discussion by mifi** [mifi/lossless-cut#1423] [General]
- 🕵️ Implement detect silence #1334
- 🕵️ Allow changing parameters of black detect #623
- 🔑 Create segments from scene changes or keyframes #1398
- 🔎 Make timeline zoom exponential
- 👬 Allow combining overlapping segments #1334
- ✅ Improve "cut finished"-dialog
- 📸 Improve snapshot capture 
  - ❓ Allow setting capture frame method #88
  - ✨ Allow changing quality #1141 #371
- 🎛️ Reorganize settings page
- 🎛️ Add setting to disable hevc #88
- 🎛️ Add setting to disable auto updates #1386 #1418
- 🎛️ Always use timecode format setting - also when exporting files
- 🖼️ Extract frames as images, new features:
  - Capture the best image every nth second
  - Capture exactly one image every nth second #1139
  - Capture exactly one image every nth frame #1139
  - Capture frames that differ the most from the previous frame
- 🖼️ Allow extract frames with timestamp files names or file numbers #1139
- ✂️ Make segments copyable #719
- fallback sanitize properly (force filenamify)
- show warning when ffmpeg vtag issue #1406 #280
- better respect "hide all notifications"
- improve export failed feedback #1409
- add more invalid chars to filename check
- always show segment name error in export page
- improve detection of mp4/mov #1353
- use adts for aac (ipod was incorrect)
- set default export save dialog path #1393
- allow aborting any operation #524
- make extract all streams only extract enabled streams #1335
- show progress in app title bar #1417
- upgrade electron to 22 (hevc suppport no longer experimental)
- add some hevc support checking #88 #1375 #1407
- fix broken progress (duration)
- fix bug with waveform #1416
- sync i18n
- workaround trashItem issue #1381
- preserve streams when fixing duration #1415
- fix broken second instance cli args #1387
- fix bool cli arg #1387
- No med spynorsk mordliste 🇳🇴

<hr /><em>This discussion was created from the release <a href='https://github.com/mifi/lossless-cut/releases/tag/v3.49.0'>3.49.0</a>.</em>

**Comment by GoubudaoFENG** [comment]
请问为什么会有解压错误Win 64

**Reply by mifi** [reply]
what is the error?

**Reply by GoubudaoFENG** [reply]
这是错误信息
正在解压文件: C:\Users\hushuhao\Downloads\Compressed\LosslessCut-win-x64.7z
正在解压到 "C:\Users\hushuhao\Downloads\Compressed\LosslessCut-win-x64\"
使用路径: 是   覆盖文件: 否
正在解压 locales
正在解压 resources
正在解压 chrome_100_percent.pak
不可纠正错误: 解压操作失败。

**Reply by mifi** [reply]
maybe try to download again. corrupt file?

**Reply by GoubudaoFENG** [reply]
好的谢谢，我再试试
